ihmc-high-level-behaviors/src/main/java/us/ihmc/behaviors/activeMapping/ContinuousHikingStateMachine/ReadyToPlanState.java

```diff
@@ -263,4 +265,15 @@ public void addWayPointPoseToList(PoseListMessage poseListMessage)
       walkToGoalWayPointPoses.add(latestWayPoint);
       debugger.publishStartAndGoalForVisualization(continuousPlanner.getStartStancePose(), latestWayPoint);
    }
+
+   /**
+    * This allows the {@link ReadyToPlanState#walkToGoalWayPointPoses} to be cleared.
+    * This empties the list so the user can place a fresh goal that Continuous Hiking will use.
+    */
+   public void clearWayPointList(Empty emptyMessage)
+   {
+      LogTools.info("Clearing waypoint list for WALK_TO_GOAL");
+      walkToGoalWayPointPoses.clear();
+      continuousHikingParameters.setEnableContinuousHiking(false);
```

Comment:
Also if we are going to clear all the goals, we probably don't want to be walking, cause we have nothing to walk towards. So make sure to stop everything.